packages/react-html/src/context.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import Manager from './manager';
+
+const {Provider, Consumer} = React.createContext<Manager>(new Manager());
+
+interface Props {
+  manager?: Manager;
+  children: React.ReactNode;
+}
+
+function HtmlManagerProvider({manager, children}: Props) {
```

Comment:
`HtmlManagerProvider` feels so java.

~Also, isn't `HtmlManager` a misnomer? It doesn't manage HTML, it manages JS serializations, no?~
I still don't like the name though. :/

Reply:
I know, could just call it `HtmlProvider` or something but w/e. It is exported from the package as just `Provider` so people can pick a name they like for it.